File: src/Lesson2/Java/Ex6.java
package Lesson2.Java;

import java.util.Scanner;

public class Ex6 {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter score: ");
        int score = scanner.nextInt();
       if (score >=90){
           System.out.println("Grade - A");
       } else if (score >=75 ) {
           System.out.println("Grade - B");
       } else if (score >=50 ) {
           System.out.println("Grade - C");
       } else if (score >=30 ) {
           System.out.println("Grade - D");
       } else
        System.out.println("F");
    }
}

File: src/Lesson2/forkotlin/Ex6.kt
package Lesson2.forkotlin

fun main (){

    println("Enter score: ")
    val score = readln().toInt()
    if (score >=90){
        println("Grade - A")
    } else if (score >=75){
        println("Grade - B")
    } else if (score >=50){
        println("Grade - C")
    } else if (score >=30){
        println("Grade - D")
    } else {
        println("Grade - F")
    }

}